internal/endtoend/testdata/between_args/mysql/go/query.sql.go

```diff
@@ -16,12 +16,41 @@ WHERE   price BETWEEN ? AND ?
 `
 
 type GetBetweenPricesParams struct {
-	Price   int32
-	Price_2 int32
+	FromPrice int32
+	ToPrice   int32
 }
 
 func (q *Queries) GetBetweenPrices(ctx context.Context, arg GetBetweenPricesParams) ([]Product, error) {
-	rows, err := q.db.QueryContext(ctx, getBetweenPrices, arg.Price, arg.Price_2)
+	rows, err := q.db.QueryContext(ctx, getBetweenPrices, arg.FromPrice, arg.ToPrice)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+	var items []Product
+	for rows.Next() {
+		var i Product
+		if err := rows.Scan(&i.ID, &i.Name, &i.Price); err != nil {
+			return nil, err
+		}
+		items = append(items, i)
+	}
+	if err := rows.Close(); err != nil {
+		return nil, err
+	}
+	if err := rows.Err(); err != nil {
+		return nil, err
+	}
+	return items, nil
+}
+
+const getBetweenPricesNamed = `-- name: GetBetweenPricesNamed :many
+SELECT  id, name, price
+FROM    products
+WHERE   price BETWEEN sqlc.arg(min_price) AND sqlc.arg(max_price)
+`
+
+func (q *Queries) GetBetweenPricesNamed(ctx context.Context) ([]Product, error) {
```

Comment:
I think your change has inadvertently broken this code. Notice that this function now has zero arguments. I think you need to keep the code that picked the number for the argument.

Reply:
Actually, that never worked. #2277 was filed about that including https://play.sqlc.dev/p/7ecf294cb7aa0b3ce1025657e091cce22970d3b4f5c4eb2baa578e5083cd4219 which shows it doesn't work.

I thought I had fixed it, but maybe I looked wrong. I'll drop the second commit [the one adding this broken test case] from this PR for now and will have a look.

Reply:
It needed an extra fix. I made that in https://github.com/kyleconroy/sqlc/pull/2373.